client/ayon_core/hosts/max/plugins/load/load_max_scene.py

```diff
@@ -67,7 +129,9 @@ def update(self, container, context):
         for prev_max_obj in prev_max_objects:
             if rt.isValidNode(prev_max_obj):  # noqa
                 rt.Delete(prev_max_obj)
-        rt.MergeMaxFile(path, quiet=True)
+        window = MaterialDupOptionsWindow(self.mtl_dup_enum)
+        window.exec_()
```

Comment:
Note: Updating should work in headless/command line mode as well without UI interference - make sure to not always force a dialog.